### ISSUE TITLE
Some transformations can't handle empty data

### ISSUE THREAD

**Issue by weidel-p** [neuromorphs/tonic#256]
Some transformations raise errors when executed on empty data. I experienced that for `ToFrame` and `Denoise` but there could be more. 

To reproduce the issue, you can try to execute the following transformation on any data:
```
transformations = tonic.transforms.Compose([tonic.transforms.DropEvent(p=1),
                                            tonic.transforms.Denoise(filter_time=10000)])
```

**Comment by biphasic** [comment]
Thanks for that Philipp. Out of interest, under what circumstances do you get rid of all the events in a recording? Would it be possible to check the length before applying a transform?
I would say if you're using a single transform only, then the check should happen in the user code, as in, if len(events) > 0, then apply the single transform.
If you build a transform pipeline with Compose and recording get filtered out completely along the way, we could add a check for the length of events [here](https://github.com/neuromorphs/tonic/blob/develop/tonic/transforms.py#L30)

**Comment by weidel-p** [comment]
We encountered this mostly when the recording we are using is finished and we read zero events for the next timestep. We do catch this case exactly the way you describe, but it can also happen in unforeseeable cases. For example when a `DropEvent` transformation unluckily removes all events. As you mention, the user can't catch the latter case.

**Comment by biphasic** [comment]
addressed in 4763042ed8ba9fa7388857110f8c0da98ed14ed0 please reopen if this is still an issue
